frontend/public/manifest.json

```diff
@@ -0,0 +1,25 @@
+{
+  "short_name": "React App",
+  "name": "Create React App Sample",
+  "icons": [
+    {
```

Comment:
should this have a better name?

Reply:
Most of these files were automatically generated (like package-lock and this one), so I didn't bother changing them.